persian/my-advanture.md

```diff
@@ -0,0 +1 @@
+چی بگم والا!!!! تموم شد دیگه. کلاس خوبی بود. مرسی udacity
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.